sql/core/src/test/resources/sql-tests/results/inline-table.sql.out

```diff
@@ -271,3 +271,11 @@ select * from values (10 + try_divide(5, 0))
 struct<col1:double>
 -- !query output
 NULL
+
+
+-- !query
+select count(distinct ct) from values now(), now(), now() as data(ct)
```

Comment:
We can add a test for `CURRENT_TIMESTAMP` and remove https://github.com/apache/spark/pull/44316/files#r1430980065